analyse_data.py

```diff
@@ -13,17 +17,49 @@
 )
 
 
-def load_build_data():
+def builds_since(builds, since):
+    """
+    Filter a list of builds to only those newer than the
+    provided timestamp.
+
+    :param float since: a UNIX timestamp.
+    """
+    # Jenkins return milliseconds since the epoch, not seconds
+    return filter(lambda b: (float(b['timestamp'])/1000) > since, builds)
```

Comment:
:question: As a general rule, we prefer datetime objects on API boundaries. No real need to do so here, but worth mentioning.